File: DesignPatternsExercises/java/ChainOfResponsibilityDesignPattern/Solution/ProductionSupport.java
public class ProductionSupport extends SupportChain {
	@Override
	public void handleIssue(IssueType issueType) {
		if(issueType == IssueType.Production) {
			System.out.println("Production support team will contact you shortly");
		} else {
			// __________(3)__________.handleIssue(issueType);
			nextSupportLink.handleIssue(issueType);
		}
	}
}

File: DesignPatternsExercises/ruby/chain_of_responsibility_design_pattern/solution/production_support.rb
require_relative 'support_chain'

class ProductionSupport < SupportChain
  def handle_issue(issue_type)
    if issue_type == :production
      puts 'Production support team will contact you shortly'
    else
      # @next_support_link.__________(3)__________(issue_type)
      @next_support_link.handle_issue(issue_type)
    end
  end
end
